iset.mm

```diff
@@ -99507,6 +99503,47 @@ the previous value and the value of the input sequence (second operand).
      version entirely.  (Contributed by Jim Kingdon, 3-Oct-2022.) $)
   df-seq3 $a |- seq M ( .+ , F ) = seq M ( .+ , F , _V ) $.
 
+  ${
+    $d .+ x y $.  $d F x y $.  $d M x y $.
+    $( Define a general-purpose operation that builds a recursive sequence
+       (i.e. a function on the positive integers ` NN ` or some other upper
+       integer set) whose value at an index is a function of its previous value
```

Comment:
Looking at the statement, I see no reason to mention `NN` ? So:
> (i.e., a function on some upper set of integers)

Reply:
I have mixed feelings about this. On the one hand, the definition is indeed not specific to `NN` at all. On the other hand, "upper integer set" is metamath-specific terminology (as far as I could tell - I tried `"upper integer set"` in two search engines, one of which had no results and one of which just had metamath on the first page) and just seems a bit awkward to me.

Given that this wording has been at https://us.metamath.org/mpeuni/df-seq.html for a long time I'm a bit reluctant to try to tackle this as part of this pull request. But I don't know, maybe we can change ``a function on the positive integers ` NN ` or some other upper integer set`` to ``a function on an upper integer set such as ` NN ` or ` NN0 ` `` without opening too many cans of worms.

Reply:
That's why I wrote "upper set of integers" above, which is standard.
Yes, something like your last proposal sounds better (with "upper set of integers").

Reply:
"upper set of integers" is no better than "upper integer set" in the searches I did.

Anyway, I'll go with the above.

Reply:
https://en.wikipedia.org/wiki/Upper_set

Reply:
Yeah, I can see where we got it from and I'm not sure there is any better terminology out there. I'm just advocating being a bit cautious, in the sense that I don't think most authors would talk about upper sets in the context of how to index a sequence.

Reply:
This strikes me as particularly bad faith. The only way one can pretend to misunderstand "upper set of integers" is by saying that the order relation on integers hasn't been specified, which is technically true, but come on.